flow/project.py

```diff
@@ -178,6 +183,19 @@ def not_(cls, condition):
                    'not_'.encode() + condition.__code__.co_code)
 
 
+def _create_all_metacondition(condition_dict, *other_funcs):
```

Comment:
This is just from unfortunate diffing. The `FlowOperation` classes are now in its place.

Reply:
Just in case this is in need of clarification, there is no material change here. This line just shows up here because it also shows up as being deleted somewhere else, but it is otherwise identical. The other changes in this file are extensive enough that Github is confused and thinks this is new code, rather than recognizing that it's been displaced by the presence of the new FlowOperation hierarchy and groups related classes.

Reply:
I feel like it should be possible to add changes in a way that this does not happen, but sure.